src/components/MxGraph.tsx

```diff
@@ -784,17 +720,15 @@ class Block {
     }
 }
 
-class Arrow {
-    public name: string;
-    public to: Block;
-
-    constructor(name: string, to: Block) {
-        this.name = name;
-        this.to = to;
-    }
+class Triple {
```

Comment:
You can actually phrase this more succinctly as

```ts
type Triple = { subject: string, predicate: string, object: string };
```

Reply:
Alternatively, leave this `class` in place and override `toString(): string` here.